src/Component.php

```diff
@@ -241,6 +243,35 @@ private function handleFor( DOMNode $node, array $data ) {
 		}
 	}
 
+	private function appendHTML( DOMNode $parent, $source ) {
+		$tmpDoc = new DOMDocument();
+		$tmpDoc->loadHTML( $source );
+		foreach ( $tmpDoc->getElementsByTagName('body')->item(0)->childNodes as $node ) {
```

Comment:
This assumes there is always a body. Is this guaranteed? Honest question – I don't know.